apps/docs/src/.vitepress/config.ts

```diff
@@ -119,12 +119,12 @@ export default defineConfig({
           base: "/development",
           items: [
             { text: "Getting Started", link: "/" },
+            { text: "Theming", link: "/theming" },
```

Comment:
The title of the markdown page is still "Theming Setup", shouldn't they align?
![image](https://github.com/SchwarzIT/onyx/assets/151019977/15f66cc5-adf3-46cb-9094-09236b0eb6a9)